.github/workflows/e2e.yml

```diff
@@ -1,10 +1,13 @@
 name: e2e
 
 on:
+  pull_request:
```

Comment:
Why do we need this trigger? e2e tests should be optional to not spend Github runners time

Reply:
Removed